activesupport/lib/active_support/cache/memory_store.rb

```diff
@@ -83,6 +84,7 @@ def decrement(name, amount = 1, options = nil)
         modify_value(name, -amount, options)
       end
 
+      # Deletes cache entires if the cache key matches a given pattern
```

Comment:
Missing "." at end of sentence. This needs some example, I will take care of that.